### ISSUE TITLE
Investigate unstable test

### ISSUE THREAD

**Issue by wxing1292** [uber/cadence#2640]
```
--- FAIL: TestESCrossDCTestSuite (66.83s)
    --- FAIL: TestESCrossDCTestSuite/TestSearchAttributes (64.50s)
        require.go:1159: 
            	Error Trace:	elasticsearch_test.go:300
            	            				elasticsearch_test.go:374
            	Error:      	Should be true
            	Test:       	TestESCrossDCTestSuite/TestSearchAttributes
```

ref: https://buildkite.com/uberopensource/cadence-server/builds/3359

**Comment by wxing1292** [comment]
solved by increasing number of retry (waiting for elastic search)

**Comment by wxing1292** [comment]
included within #2642
